projects/fsastorefrontstyles/scss/overrides/homepage.scss

```diff
@@ -65,6 +65,44 @@
     }
   }
 
+  // Need analysis entry point section
+  .Section3 {
+    .homepage-questionnaire {
+      border-radius: 2.5rem;
+      background-color: var(--cx-color-light);
+      font-size: var(--cx-h5-font-size);
+      cx-generic-link {
+        display: inline-block;
+        padding: 0 65px;
+      }
+      cx-generic-link a {
+        font-weight: var(--cx-font-weight-semi);
+      }
+      &::before,
+      &::after {
+        @include vertical-center;
+        font-family: 'Font Awesome 5 Free', Arial;
```

Comment:
Reported as a major bug by sonar. Apparently, the browser requires that font declarations should contain at least one generic font family like sans-serif. 
Not sure if it should be fixed though.

Reply:
Yes, I saw that, so I tried with `Arial `. I'll try also with `sans-serif`.